Library/Homebrew/test/support/lib/config.rb

```diff
@@ -15,21 +15,21 @@
 
 # Paths pointing into the Homebrew code base that persist across test runs
 HOMEBREW_LIBRARY_PATH  = Pathname.new(File.expand_path("../../../..", __FILE__))
-HOMEBREW_SHIMS_PATH    = HOMEBREW_LIBRARY_PATH.parent+"Homebrew/shims"
+HOMEBREW_SHIMS_PATH    = HOMEBREW_LIBRARY_PATH.parent.join("Homebrew", "shims")
```

Comment:
Please don't use `.join` for paths in Homebrew code. If you want to change this use `HOMEBREW_LIBRARY_PATH.parent/"Homebrew/shims"`. Feel free to remove uses of `join` here, though.